examples/ceres-solver-suitesparse-dynLAPACK/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+set(TESTING_CONFIG_OPT FILEPATH "${CMAKE_CURRENT_LIST_DIR}/config.cmake")
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ceres-solver)
+# build LAPACK as shared library
```

Comment:
I don't think these comments are needed. You can see from `TESTING_CONFIG_OPT` that we are loading custom `config.cmake`, and in `config.cmake` it's clear what is happenning.